platform/linuxbsd/detect.py

```diff
@@ -190,9 +188,7 @@ def configure(env):
             env.Append(CCFLAGS=["-flto"])
             env.Append(LINKFLAGS=["-flto"])
 
-        if not env["use_llvm"]:
-            env["RANLIB"] = "gcc-ranlib"
-            env["AR"] = "gcc-ar"
+    print(env["lto"])
```

Comment:
Is this wanted? It looks like a debug statement to me.

Reply:
Indeed, thanks. I'll remove it.